pyanaconda/ui/cockpit/anaconda-webui/src/InstallationDestination.jsx

```diff
@@ -35,7 +159,14 @@ export const InstallationDestination = () => {
               title='Installation destination'
             />
             <PageSection>
-                Not implemented
+                <Form isHorizontal>
+                    <Hint>
+                        <HintBody>
+                            Select the device(s) you would like to install to. They will be left untouched until you click on the main menu's 'Begin installation' button.
```

Comment:
Translations is the first topic I want to deal with once the draft PR is in. It's really simple, it's handled by cockpit, not patternfly. We need to pass strings for translation as _("I need to translate this") rather than "I need to translate this"